### ISSUE TITLE
[BLOCKING] AppBuilder().automaticBuildAndRun broken on windows

### ISSUE THREAD

**Issue by Suyashtnt** [mandarineorg/mandarinets#232]
**[Details]**
the AppBuilder().automaticBuildAndRun method is brokn on windows

**[Stacktrace]**
```
PS C:\Users\Home\Documents\deno-tutorial> deno run --allow-net --allow-read --allow-write --allow-env --allow-run -c tsconfig.json --unstable  src/mandarine/app.ts
Download https://deno.land/x/mandarinets@v2.2.0/mod.ts
error: Unsupported scheme "c" for module "c:UsersHomeDocumentsdeno-tutorialsrcmandarineapiapi.controller.ts". Supported schemes: [
    "http",
    "https",
    "file",
]
Imported from "file:///C:/Users/Home/Documents/deno-tutorial/.mandarine_target/target.ts:2"
```

**[Scenario]**
1) Create a new mandarine project in windows
2) Use the automatic app builder
3) run it

**[Expected behavior]**
It would start

**[Screenshots]**
![https://cdn.discordapp.com/attachments/727894026606608414/772003980452560966/unknown.png]

**[OS]**
Windows 10

**[Environment]**
 - Mandarine version: 2.2.0
 - Deno version: 1.4.6
 - Deno v8 version: 8.7.220.3
 - Deno typescript version: 4.0.3

**[Additional]**
Took a look in `target.ts` and it seems to be that the directories are wrong

**Comment by andreespirela** [comment]
Fix released in 2.2.1 https://github.com/mandarineorg/mandarinets/releases/tag/v2.2.1

**Comment by andreespirela** [comment]
Seems like this hasn’t been fixed for some windows. Reopining
